src/catkin_pkg/package_templates.py

```diff
@@ -294,10 +294,12 @@ def _create_targetlib_args(package_template):
 
 
 def _create_include_macro(package_template):
-    result = '# include_directories(include)'
+    result = '#  include'
     includes = []
     if package_template.catkin_deps:
         includes.append('${catkin_INCLUDE_DIRS}')
+    else:
+        includes.append('#  ${catkin_INCLUDE_DIRS}')
```

Comment:
This certainly works. In general I try to avoid repetition whenever possible. Can you update this block to something like this:

```
include = '${catkin_INCLUDE_DIRS}'
if not package_template.catkin_deps:
    include = '# ' + include
includes.append(include)
```